src/core/Tabs/index.tsx

```diff
@@ -0,0 +1,102 @@
+import React, { Fragment } from 'react';
+import useStyles from './styles';
+import PropTypes from 'prop-types';
+import Tab from '@material-ui/core/Tab';
+import Box from '@material-ui/core/Box';
+import Tabs from '@material-ui/core/Tabs';
+import AppBar from '@material-ui/core/AppBar';
+import Typography from '@material-ui/core/Typography';
+import { withStyles } from '@material-ui/core/styles';
+
+const TabPanel = (props: any) => {
```

Comment:
- props should not have any type
- interface should be there for props
- props should be destructured in parameters